common/deftype/CMakeLists.txt

```diff
@@ -52,7 +52,14 @@ endif()
 ADD_DEFINITIONS( -D_USRDLL -DDEFTYPE_EXPORTS )
 
 HPCC_ADD_LIBRARY( deftype SHARED ${SRCS} )
-install ( TARGETS deftype RUNTIME DESTINATION ${EXEC_DIR} LIBRARY DESTINATION ${LIB_DIR} )
+
+if(NOT PLUGIN)
+     install(
+          TARGETS deftype
+          RUNTIME DESTINATION ${EXEC_DIR}
+          LIBRARY DESTINATION ${LIB_DIR})
+endif()
```

Comment:
This block is a guard that I added to several of the subdirectories. All the plugins were adding extra files that weren't needed from these directories because of the additions to the if(PLUGIN) add_subdirectory() portion of the top level cmake when we added NLP.

This mimics the behavior we have in jlib already, and a few other directories we were building to support plugin development.